packages/manager/modules/pci/src/projects/project/private-networks/add/translations/Messages_fr_FR.json

```diff
@@ -13,9 +13,9 @@
   "pci_projects_project_network_private_create_name": "Nom du réseau privé",
   "pci_projects_project_network_private_create_name_placeholder": "Saisir un nom de réseau privé",
   "pci_projects_project_network_private_create_public_gateway": "Créez une Gateway et connectez-vous au réseau privé",
-  "pci_projects_project_network_private_create_public_gateway_decription_1": "Cette option permet d'accéder à Internet à partir d'instances de réseau privé. Gateway est automatiquement annoncée comme passerelle par défaut dans le nouveau réseau privé. Pour en savoir plus, <a href=\"{{ guideLink }}\" target=\"_blank\">cliquez ici</a>. Cette option est en supplément (voir le récapitulatif ci-dessous pour plus de détails).",
+  "pci_projects_project_network_private_create_public_gateway_decription_1": "Cette option permet d'accéder à Internet à partir d'instances de réseau privé. Gateway est automatiquement annoncée comme passerelle par défaut dans le nouveau réseau privé. Pour en savoir plus, <a href=\"{{ guideLink }}\" target=\"_blank\" data-track-on=\"click\" data-track-type=\"action\" data-track-name=\" {{ trackLabel }} \" >cliquez ici</a>. Cette option est en supplément (voir le récapitulatif ci-dessous pour plus de détails).",
```

Comment:
I think is better if we remove the html code in translation file.

Reply:
I am not sure if we can split the text to be shown as link into a different sentence. So, I cannot remove the HTML from translation files.

Reply:
I think that data-translate works on <a></a>.

Reply:
I need to display the text `cliquez ici` as link. And since this is a part of the sentence, I am afraid it cannot be put as a separate sentence. And there is no possibility to display a part of sentence as a link apart from including the `<a>` in the translation files.

Reply:
Indeed, I didn't notice that it was in the middle of a sentence.